workflows/api/Workflow.Samples/Execute.cs

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START workflows_execute_without_arguments]
+
+using Google.Cloud.Workflows.Common.V1;
+using Google.Cloud.Workflows.Executions.V1;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class ExecuteWorkflowSample
+{
+    /// <summary>
+    /// Execute a workflow and return the execution operation.
+    /// </summary>
+    /// <param name="projectID">Your Google Cloud Project ID.</param>
+    /// <param name="locationID">The region where your workflow is located.</param>
+    /// <param name="workflowID">Your Workflow ID.</param>
+    public async Task<Execution> ExecuteWorkflow(
+        string projectId = "YOUR-PROJECT-ID",
+        string locationID = "YOUR-LOCATION-ID",
+        string workflowID = "YOUR-WORKFLOW-ID")
+    {
+        // Initialize the client.
+        ExecutionsClient client = await ExecutionsClient.CreateAsync();
+
+        // Build the parent location path.
+        WorkflowName parent = new WorkflowName(projectId, locationID, workflowID);
+
+        // Create an execution request.
+        CreateExecutionRequest createExecutionRequest = new CreateExecutionRequest
+        {
+            ParentAsWorkflowName = parent,
+        };
+
+        // Execute the operation.
+        Execution execution = await client.CreateExecutionAsync(createExecutionRequest);
+        Console.WriteLine("- Execution started...");
+
+        TimeSpan backoffDelay = TimeSpan.FromSeconds(1);
+
+        // Keep polling the state until the execution finishes, using exponential backoff.
+        while (execution.State == Execution.Types.State.Active)
+        {
+            await Task.Delay(backoffDelay);
+            // Exponential delay by doubling the current value (capped in 16 seconds).
+            if (backoffDelay < TimeSpan.FromSeconds(16))
+            {
+                backoffDelay *= 2;
+            }
```

Comment:
I think you can do something like:
```suggestion
            backoffDelay = backoffDelay * 2 as TimeSpan maybeNewBackoff < maxBackoffDelay
                ? maybeNewBackoff
                : maxBackoffDelay;
```